execa.js

```diff
@@ -0,0 +1,31 @@
+// Mini wrapper around child_process to make it behave a little like execa
+
+const {promisify} = require('util');
+const childProcess = require('child_process');
+
+const execFile = promisify(childProcess.execFile);
+
+/**
+@param  {string} command
+@param  {string[]} args
+
+@returns Promise<import('child_process').ChildProcess>
+*/
```

Comment:
I added these to help their use in editors that show types

<img width="556" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/110382173-269c0500-8020-11eb-843c-39e709e0f126.png">